config/locales/en.yml

```diff
@@ -15,14 +15,48 @@ en:
   category_records:
     form:
       existing_categories: "Existing categories"
+    new:
+      title: "New category"
   defaults:
     destroy_confirm: "Are you sure?"
     destroy_link: "Destroy"
     edit_link: "Edit"
+    back_link: "Back"
     new_link: "New %{model}"
```

Comment:
You could use this syntax in more places to reduce duplication. For example, right now we have separate translations for "donations.new.title", "recipients.new.title", etc. but they are all quite similar.

Reply:
Same goes for edit.title